controllers/topics/inviteMembers.js

```diff
@@ -92,12 +92,14 @@ module.exports = async (req, res) => {
 
         const payload = {
           notification: {
-            title: invitations_msg,
+            title: `#${topics?.name}`,
+            body: invitations_msg,
             image: user.profile_pic_path
           },
           data: {
             topic_id,
-            type: 'topic'
+            type: 'topic',
+            topic_name: topics?.name
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes look good, but there's a potential issue here. If `topics` is `null` or `undefined`, the `title` and `data.topic_name` fields will be set to `undefined`. This could lead to unexpected behavior when the payload is processed. It would be better to provide default values for these fields.

```diff
-            title: `#${topics?.name}`,
+            title: `#${topics?.name || 'unknown'}`,
...
-            topic_name: topics?.name
+            topic_name: topics?.name || 'unknown'
```
This way, if `topics` is `null` or `undefined`, the `title` and `data.topic_name` fields will be set to `'unknown'`, which should be handled appropriately by the code that processes the payload.


<!-- This is an auto-generated comment by OSS CodeRabbit -->